src/Provider/AbstractProvider.php

```diff
@@ -31,7 +31,9 @@
 
     public $responseType = 'json';
 
-    public $headers = null;
+    public $headers = [];
+
+    public $authorizationHeader;
```

Comment:
What is the impact if I merge this in now and tag a release? How badly will providers break if they are using oauth2-client `~0.9` (as they would be automatically upgraded to `0.10` if using versioning like this)?